phys/module_mp_p3.F

```diff
@@ -460,10 +567,18 @@ SUBROUTINE p3_init(lookup_file_dir,nCat)
 
 !.......................................................................
 
- print*, '   P3_INIT DONE.'
- print*
+ if (procnum == 0) then
```

Comment:
We probably have physics codes with print statements that say "I'm starting". These should probably be wrf_debug statements, with a non-zero triggering value.